File: Lua/bee 1042.lua
function bubbleSort(arr)
    for i=1,#arr-1 do
        for j=1,#arr-1 do
            if arr[j] > arr[j+1] then
                local aux = arr[j]
                arr[j] = arr[j+1]
                arr[j+1] = aux
            end
        end
    end
    return arr
end

local a = tonumber(io.read("*n"))
local b = tonumber(io.read("*n"))
local c = tonumber(io.read("*n"))

local arr = {a,b,c}
local arr_ordered = bubbleSort(arr)

print(string.format("%d\n%d\n%d\n", arr_ordered[1], arr_ordered[2], arr_ordered[3]))
print(string.format("%d\n%d\n%d", a, b, c))

File: Rust/bee_1042.rs
use std::io::{self, Read};

fn bubble_sort(v: &mut [i32]) {
    for _ in 0..v.len() - 1 {
        for j in 0..v.len() - 1 {
            if v[j + 1] < v[j] {
                let aux = v[j];
                v[j] = v[j + 1];
                v[j + 1] = aux;
            }
        }
    }
}

fn main() {
    let mut input = String::new();
    io::stdin().read_to_string(&mut input).unwrap();
    let mut valores: Vec<i32> = input
        .split_whitespace()
        .map(|x| x.parse().unwrap())
        .collect();

    let valores_desordenados = valores.clone();

    bubble_sort(&mut valores);

    for valor in valores {
        println!("{}", valor);
    }

    println!();

    for valor in valores_desordenados {
        println!("{}", valor);
    }
}
